Remove mention of the "view" alias from the manpage.
It won't be installed as it would conflict with vi packages.

Remove mention of the "update-mime" tool from the manpage.
It won't be installed as it is specific to Debian packages.

--- run-mailcap.man.orig	2014-12-28 07:06:43.000000000 +0100
+++ run-mailcap.man	2015-01-14 20:40:31.550692133 +0100
@@ -4,7 +4,7 @@
 .\"
 .TH RUN-MAILCAP 1 "1st Jan 2008" "Debian Project" "Run Mailcap Programs"
 .SH NAME
-run-mailcap, view, see, edit, compose, print \- execute programs via entries in the
+run-mailcap, see, edit, compose, print \- execute programs via entries in the
 .I mailcap
 file
 .SH SYNOPSIS
@@ -81,7 +81,6 @@
 Displays the found command without actually executing it.
 .SH "SEE ALSO"
 .BR file "(1)"
-.BR update-mime "(8)"
 .SH AUTHOR
 .B run-mailcap
 (and its aliases) was written by Brian White <bcwhite@pobox.com>.
